README.md

```diff
@@ -107,7 +110,7 @@ Example command:
 Clone this repository to your desired folder:
 
 ```sh
-  git clone git@github.com:NicolasAndrehh/Webpack-Boilerplate.git
+  git clone git@github.com:NicolasAndrehh/JavaScript-Group-Capstone.git
 ```
 
 ### Install
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** I think it would be better if you add a command of how to access the cloned directory after a User has cloned it locally to make your documentation more descriptive. i.e.
```
cd JavaScript-Group-Capstone 
```